test/integration/cart_integration_test.rb

```diff
@@ -104,6 +104,75 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
     assert_equal showcart_path, current_path
   end
 
+  test 'a user can edit the quantity of an item in their cart' do
+    coffee = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.